packages/fiori/src/themes/Bar.css

```diff
@@ -0,0 +1,65 @@
+:host {
+	background-color: var(--sapPageHeader_Background);
+	height: var(--_ui5_bar_base_height);
+	width: 100%;
+	box-shadow: inset 0 -0.0625rem var(--sapPageHeader_BorderColor);
+	display: block;
+	margin-bottom: 0.5rem;
+	margin-top: 0.5rem;
+}
+
+.ui5-bar-root {
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	height: inherit;
+	width: inherit;
+	background-color: inherit;
+	box-shadow: inherit;
+	border-radius: inherit;
+}
+
+.ui5-bar-root .ui5-bar-startcontent-container {
+	padding-left: 0.5rem;
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+}
+
+.ui5-bar-root .ui5-bar-endcontent-container {
+	padding-right: 0.5rem;
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+}
+.ui5-bar-root .ui5-bar-midcontent-container {
+	padding-left: 0.5rem;
+	padding-right: 0.5rem;
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+}
+
+:host([design="Footer"]){
+	background-color: var(--sapPageFooter_Background);
+	border-top: 0.0625rem solid var(--sapPageFooter_BorderColor);
+	box-shadow: none;
+}
+
+:host([design="SubHeader"]){
+	height: var(--_ui5_bar_subheader_height);
+}
+
+:host([design="FloatingFooter"]){
+	border-radius: var(--sapElement_BorderCornerRadius);
+	background-color: var(--sapPageFooter_Background);
+	box-shadow: var(--sapContent_Shadow1);
+	border: none;
+}
+
+::slotted([slot="middleContent"]),
+::slotted([slot="endContent"]),
+::slotted([slot="startContent"]){
+	display: flex;
+	align-items: center;
```

Comment:
When the children are directly added, what is the case you have to advertise, 
add margin `margin: 0 0.25rem;` so there is space between the elements in the bat out of the box.

Reply:
Isn't it ::slotted(*) the same selector only shorter? Didn't see slot contents that doesn't need those properties.